### PR TITLE
Add devicemgt_admin jax-rs and removed mdm-admin jax-rs

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: modules/integration/tests-iot-web-ui/src/test/resources/testng.xml [hunk 1]
```diff
@@ -28,17 +28,19 @@
         <classes>
             <class name="org.wso2.carbon.iot.integration.web.ui.test.login.LoginFormValidationTest"/>
             <class name="org.wso2.carbon.iot.integration.web.ui.test.login.LoginTest"/>
-            <class name="org.wso2.carbon.iot.integration.web.ui.test.user.RegistrationFormValidationTest"/>
+            <!--<class name="org.wso2.carbon.iot.integration.web.ui.test.user.RegistrationFormValidationTest"/>-->
```
Comment (DilanUA) [inline modules/integration/tests-iot-web-ui/src/test/resources/testng.xml:31]: remove unnecessary commented lines.

File: modules/integration/tests-iot-web-ui/src/test/resources/testng.xml [hunk 2]
```diff
@@ -28,17 +28,19 @@
...
             <class name="org.wso2.carbon.iot.integration.web.ui.test.user.AddUserFormValidationTest"/>
-            <class name="org.wso2.carbon.iot.integration.web.ui.test.user.AdminFunctionsTest"/>
+            <!--<class name="org.wso2.carbon.iot.integration.web.ui.test.user.AdminFunctionsTest"/>-->
+            <!--
             <class name="org.wso2.carbon.iot.integration.web.ui.test.samples.SampleInstallationTest"/>
             <class name="org.wso2.carbon.iot.integration.web.ui.test.samples.SampleInstallationVerification"/>
             <class name="org.wso2.carbon.iot.integration.web.ui.test.samples.SampleEnrollmentTest"/>
             <class name="org.wso2.carbon.iot.integration.web.ui.test.samples.SampleEnrolmentVerificationTest"/>
             <class name="org.wso2.carbon.iot.integration.web.ui.test.samples.SampleFunctionalityTest"/>
+            -->
```
Comment (DilanUA) [inline modules/integration/tests-iot-web-ui/src/test/resources/testng.xml:43]: remove unnecessary commented lines.

File: modules/integration/tests-iot-web-ui/src/test/resources/testng.xml (hunk 1 above)
Comment (charithag) [inline modules/integration/tests-iot-web-ui/src/test/resources/testng.xml:31]: These lines were commented to skip tests temporary as we are modifying the areas which related to those tests. They need to uncomment once we have completed the changes.

File: modules/integration/tests-iot-web-ui/src/test/resources/testng.xml (hunk 2 above)
Comment (charithag) [inline modules/integration/tests-iot-web-ui/src/test/resources/testng.xml:43]: These lines were commented to skip tests temporary as we are modifying the areas which related to those tests. They need to uncomment once we have completed the changes.

**Pull request merged**